DESCRIPTION

```diff
@@ -24,9 +24,12 @@ Imports:
     yaml
 Suggests:
     knitr (>= 1.8),
+    downlit,
+    pkgdown,
     rmarkdown,
     testthat
 Enhances: shiny (>= 1.1)
 URL: https://github.com/ramnathv/htmlwidgets
 BugReports: https://github.com/ramnathv/htmlwidgets/issues
 RoxygenNote: 7.1.1
+Remotes: dmurdoch/downlit@rglpatch, dmurdoch/pkgdown@rglpatch
```

Comment:
Have these PRs been merged? If so, the Remotes can be updated to point to the main repositories.